src/Connector.AzureAIStudio/Connector/AzureAIStudioExportEntitiesJob.cs

```diff
@@ -18,7 +18,7 @@ public AzureAIStudioExportEntitiesJob(
     {
     }
 
-    protected override string GetOutputFileName(Guid streamId, DateTime asOfTime, string outputFormat)
+    protected override string GetOutputFileName(IDataLakeJobData configuration, Guid streamId, string containerName, DateTime asOfTime, string outputFormat)
```

Comment:
I think same thing should be done for DataBricks & Synapse Data Engineering too?